src/foam/dao/AuthenticatedDAO.java

```diff
@@ -1,17 +1,20 @@
 /**
  * @license
- * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
```

Comment:
According to Google lawyers, you just leave the copyright as the year the file was first created.